infra/modules/gcp-sa-auth-key-aws-secret/main.tf

```diff
@@ -27,4 +27,10 @@ resource "aws_ssm_parameter" "value" {
   type        = "SecureString"
   description = "Key for gcp service account ${var.service_account_id}"
   value       = google_service_account_key.key.private_key
+
+  lifecycle {
+    ignore_changes = [
+      tags,
```

Comment:
:+1: just saw this

Reply:
yeah - the `tags` isn't such a big deal.  It just seems that many customers use `tags` in practice, so after they run our terraform stuff they go through and tag stuff (sometimes manually, or with asset discovery scripts - I'm not really clear) - but either way I don't want to overwrite their changes - nor to I want to add the extensibility to our scripts to pass through all possible tags they might want to encode.